middle_end/flambda2/simplify/flow/flow_acc.ml

```diff
@@ -445,6 +445,12 @@ let extend_args_with_extra_args (t : T.Acc.t) =
                   rewrite_ids)
             elt.apply_cont_args
         in
+        let apply_cont_args =
```

Comment:
Could you add a comment here, resuming what you explain in the description of the PR ?

Reply:
Btw, we should try and add some comments in flow<_analysis, at least to explain what each piece of code is supposed to do: re-reading the code is slightly painful at the moment, even though I wrote most of it, so I'll see if I have time to add some doc.

Reply:
Also, if we want to, we could try and fuse the `filter` added here to the `mapi` above, with some `Map.filter_mapi` (though I doubt that function exists) ?